crates/evm/src/executor/backend/mod.rs

```diff
@@ -1624,7 +1625,7 @@ impl BackendInner {
 
     fn next_id(&mut self) -> U256 {
         let id = self.next_fork_id;
-        self.next_fork_id += U256::one();
+        self.next_fork_id += U256::from(1);
```

Comment:
::ONE works i think

Reply:
seems `ZERO`'s the only constant available: https://docs.rs/ruint/1.10.1/ruint/struct.Uint.html#associatedconstant.ZERO